packages/lib/src/core/helpers/stylesMigrator.js

```diff
@@ -0,0 +1,35 @@
+import utils from '../../common/utils/index';
```

Comment:
@yhattav 
This will later be inject as an argument? From the wix-private repos?